src/components/Dialogs/FileBrowser/ImageSave/ImageSaveComponent.tsx

```diff
@@ -152,9 +152,9 @@ export class ImageSaveComponent extends React.Component {
                   })
                 : [];
         // Calculate a small step size
-        const numChannels = activeFrame.numChannels;
-        const min = activeFrame.channelValueBounds?.min;
-        const max = activeFrame.channelValueBounds?.max;
+        const numChannels = activeFrame?.numChannels;
+        const min = activeFrame?.channelValueBounds?.min;
```

Comment:
if `activeChannel` is undefined, then `min` and `max` are both undefined. So `delta` will be `NaN`. Perhaps we should make min and max default to `0` and `1` using the `??` operator?

Unless passing `NaN` as step size works fine?

Reply:
`NaN` works fine because all elements is hidden when no activeFrame, but I think it's safer to set to 0 and 1. will change this part.